src/styles/calendar.scss

```diff
@@ -0,0 +1,46 @@
+.calendar {
+  width: $width;
+  display: flex;
+  flex-wrap: wrap;
+  gap: 1px;
+  padding-top: 20px;
+
+  &__day {
+    width: $size;
+    height: $size;
+    border: 1px solid black;
+    background-color: #eee;
+
+    display: flex;
+    justify-content: center;
+    align-items: center;
+
+    &:hover {
+      cursor: pointer;
+      background: #f698a9;
+      transform: translateY(-20px);
+      transition: 0.5s;
+    }
+
+    @for $index from 1 through 31 {
+      &:nth-child(#{$index})::before {
+        content: "#{$index}";
+        font-family: Arial, Helvetica, sans-serif;
+        font-size: 30px;
```

Comment:
You better add a variable for text size. This will help you if you decide to change the width of the "day" and make it smaller but the size of the text will remain the same "big"